frontend/src/screens/settings/AlbyAccount.tsx

```diff
@@ -0,0 +1,93 @@
+import { ExitIcon } from "@radix-ui/react-icons";
+import { ExternalLinkIcon } from "lucide-react";
+
+import ExternalLink from "src/components/ExternalLink";
+import SettingsHeader from "src/components/SettingsHeader";
+import {
+  Card,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "src/components/ui/card";
+import { useToast } from "src/components/ui/use-toast";
+import { useCSRF } from "src/hooks/useCSRF";
+import { useInfo } from "src/hooks/useInfo";
+import { request } from "src/utils/request";
+
+export function AlbyAccount() {
+  const { data: csrf } = useCSRF();
+  const { toast } = useToast();
+  const { mutate: refetchInfo } = useInfo();
+
+  const unlink = async () => {
+    if (
+      !confirm(
+        "Please log out at https://getalby.com, then click ok to continue."
+      )
+    ) {
+      return;
+    }
+
+    if (
+      !confirm("Are you sure you want to change the Alby Account for your hub?")
+    ) {
+      return;
+    }
+
+    try {
+      if (!csrf) {
+        throw new Error("No CSRF token");
+      }
+      await request("/api/alby/unlink-account", {
+        method: "POST",
+        headers: {
+          "X-CSRF-Token": csrf,
+          "Content-Type": "application/json",
+        },
+      });
+      await refetchInfo();
```

Comment:
navigate to the URL instead of refetching the info, and pass a query param so we can include `force_login` in the auth URL